### PR TITLE
Add ability to associate attendees checking in with kiosks.

### PR DESCRIPTION
**Pull request by leizleiz**

### CODE REVIEW

File: scalereg/reg6/views.py [hunk 1]
```diff
@@ -23,6 +23,8 @@
 
 PGP_KEY_QUESTION_INDEX_OFFSET = 3
 
+KIOSK_AGENT = 'CHANGE_ME:'
```
Comment (irabinovitch) [inline scalereg/reg6/views.py]: Should this come from settings.py instead?

File: scalereg/scale_templates/reg6/reg_kiosk.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
     <table border="0" width="100%">
     <tr>
     <td align="left">
-    <p>Registration</p>
+    <p>SCALE Registration (Station {{ agent }})</p>
```
Comment (irabinovitch) [inline scalereg/scale_templates/reg6/reg_kiosk.html]: Pull event name from settings?

File: scalereg/reg6/views.py (hunk 1 above)
Comment (leizleiz) [reply scalereg/reg6/views.py]: Done.

File: scalereg/scale_templates/reg6/reg_kiosk.html (hunk 2 above)
Comment (leizleiz) [reply scalereg/scale_templates/reg6/reg_kiosk.html]: I left out "SCALE" from the line. Parameterizing the event name in various templates would be a PR all on its own.

**Pull request merged**